pkg/providers/amifamily/suite_test.go

```diff
@@ -184,20 +174,21 @@ var _ = Describe("AMIProvider", func() {
 		Expect(amis).To(HaveLen(1))
 	})
 	It("should succeed to resolve AMIs (Windows2022)", func() {
-		nodeClass.Spec.AMIFamily = &v1beta1.AMIFamilyWindows2022
+		nodeClass.Spec.AMISelectorTerms = []v1beta1.AMISelectorTerm{{Alias: "windows2022@latest"}}
 		awsEnv.SSMAPI.Parameters = map[string]string{
 			fmt.Sprintf("/aws/service/ami-windows-latest/Windows_Server-2022-English-Core-EKS_Optimized-%s/image_id", version): amd64AMI,
 		}
 		amis, err := awsEnv.AMIProvider.List(ctx, nodeClass)
 		Expect(err).ToNot(HaveOccurred())
 		Expect(amis).To(HaveLen(1))
 	})
-	It("should succeed to resolve AMIs (Custom)", func() {
-		nodeClass.Spec.AMIFamily = &v1beta1.AMIFamilyCustom
-		amis, err := awsEnv.AMIProvider.List(ctx, nodeClass)
-		Expect(err).ToNot(HaveOccurred())
-		Expect(amis).To(HaveLen(0))
-	})
+	// TODO: (jmdeal@) Revisit this test, what is it meant to be doing
```

Comment:
Seems like the test is just making sure that we don't return any default AMIs for custom AMI family, right, but that it doesn't succeed?

Reply:
Ya, I left this comment because I wasn't sure it would still apply, and it really doesn't because there shouldn't be any way to do this anymore. You can only use the `Custom` family when no alias is specified, but there must be some other selector term.